interfaces/Portal/src/app/program/export-select/export-select.component.ts

```diff
@@ -7,13 +7,16 @@ import { ProgramPhase } from '../../models/program.model';
   templateUrl: './export-select.component.html',
   styleUrls: ['./export-select.component.css'],
 })
-export class ExportSelectComponent implements OnInit {
+export class ExportSelectComponent implements AfterViewChecked {
   @Input()
   public programId: number;
 
   @Input()
   public thisPhase: ProgramPhase;
 
+  @Input()
+  public showValidation: boolean;
+
   public isPopoverOpen = false;
```

Comment:
```suggestion
```

This property isn't used anymore, right?

Reply:
it is right? When I search all for it, it clearly seems to be used.